src/languages/es.ts

```diff
@@ -324,6 +324,9 @@ export default {
         action: 'Acción',
         expenses: 'Gastos',
         tax: 'Impuesto',
+        shared: 'Compartido',
```

Comment:
```suggestion
        shared: 'Compartidos',
```

Reply:
Bump @jnowakow. I think we should change this to plural since all other ones are plural too